docs/docs/07-forms.md

```diff
@@ -35,7 +35,7 @@ Like all DOM events, the `onChange` prop is supported on all native components a
 
 ## Controlled Components
 
-An `<input>` with `value` set is a *controlled* component. In a controlled `<input>`, the value of the rendered element will always reflect the `value` prop. For example:
+An `<input>` with a `value` prop is a *controlled* component:
```

Comment:
Yep, sounds good to me.

We try to use the word 'prop' across the docs when referring to properties on elements, so the line would become:
> A **controlled** `<input>` has `value` prop. Rendering a controlled `<input>` will reflect the value of the `value` prop.